unime/src/i18n/de-DE/index.ts

```diff
@@ -291,6 +291,13 @@ const de = {
       OLDEST: 'Älteste zuerst',
     },
   },
+  DOMAIN_LINKAGE: {
+    TITLE: 'Verifiziert',
+    SUCCESS: 'UniMe konnte die Identität erfolgreich verifizieren, um dir einen sicheren Login zu ermöglichen.',
+    FAILURE: 'UniMe konnte die Verknüpfung der Identität mit der Domain nicht überprüfen.',
+    UNKNOWN: 'UniMe konnte keinen Nachweis über die verbundene Identität der Domain finden.',
+    CAUTION: 'Fahre mit Vorsicht fort!',
```

Comment:
Ich würde die Infinitiv-Form benutzen: "Mit Vorsicht fortfahren!" oder "Vorsicht walten lassen!".

Reply:
Okay, updated to use "Mit Vorsicht fortfahren!" 👍